admin/package.json

```diff
@@ -30,6 +30,7 @@
     "express": "~4.15.2",
     "express-breadcrumbs": "0.0.2",
     "express-busboy": "^6.0.1",
+    "express-enforces-ssl": "^1.1.0",
```

Comment:
If adding new packages please also update the yarn lock file (for travis)

Reply:
thanks for the heads up, adjustment incoming...